.travis.yml

```diff
@@ -0,0 +1,17 @@
+language: python
```

Comment:
Not sure what the deal is with this package, but we are slowly moving to using GH actions as Travis now offers limited and spotty support for open source projects. Just a heads up.

Reply:
Awesome, thank you, I see no reason to keep using Travis then. fe05c9d221 , 4673e7b1e, and e24a7084b81 add an actions workflow that seems to be working.

Reply:
Looks good.

Reply:
👍